Emby.Server.Implementations/Updates/InstallationManager.cs

```diff
@@ -349,26 +342,26 @@ private async Task InstallPackageInternal(PackageVersionInfo package, Cancellati
             // Do plugin-specific processing
             if (plugin == null)
             {
-                _logger.LogInformation("New plugin installed: {0} {1} {2}", package.name, package.versionStr ?? string.Empty, package.classification);
+                _logger.LogInformation("New plugin installed: {0} {1} {2}", package.name, package.version);
```

Comment:
what if `package.version` is `null`?

Reply:
It shouldn't ever be the case, or my IDE would have thrown an error.